zipkin/zipkin/run.sh

```diff
@@ -4,4 +4,10 @@ if [ -f ".${STORAGE_TYPE}_profile" ]; then
   source ${PWD}/.${STORAGE_TYPE}_profile
 fi
 
-exec java ${MODULE_OPTS} ${JAVA_OPTS} -cp . org.springframework.boot.loader.PropertiesLauncher
+# Use main class directly if there are no modules, as it measured 14% faster from JVM running to available
+# verses PropertiesLauncher when using Zipkin was based on Spring Boot 2.1
+if [[ -z "$MODULE_OPTS" ]]; then
+  exec java ${JAVA_OPTS} -cp .:$(ls BOOT-INF/lib/*.jar|tr '\n' ':')BOOT-INF/classes zipkin.server.ZipkinServer
```

Comment:
Assuming you tried just the wildcard and it didn't work

Reply:
yeah heh I wanted it to!